### ISSUE TITLE
collecting data project / rubric

### ISSUE THREAD

**Issue by retabak** [bootstrapworld/curriculum#865]
I'm playing around with the collecting data project before starting on a rubric. If I were doing this project with my students, I would want  / need more structure - so I created some. 

The[ teacher directions](https://docs.google.com/document/d/1G1yLyGoY-5COdNLKvGnzWwF3KqBg_lBoasQVuwtfzzo/edit?usp=sharing) are now more specific, and there's a [student-facing page](https://docs.google.com/document/d/1Xb-9XhNre8GZLSae5TMp1gI3KkVtpM4aryXYBa5Ekxk/edit?usp=sharing) - because it was unclear to me before what exactly kids were supposed to submit. (These only exist as google docs at the moment.)

Let me know what you think. If this looks okay, I can start on a rubric (which I'll share here for review later!).

**Comment by retabak** [comment]
And here's a rubric draft! Will asciidocify once I've got feedback.

https://docs.google.com/document/d/1pPr1OyFcUMWwZ6k5jlLmC4IVatr2ZOQwLIshkOdBtP8/edit?usp=sharing

**Comment by flannery-denny** [comment]
@retabak Thanks for taking a pass at this. I'm not sure exactly what @schanzer's vision for this project was, but, in my classroom I would probably expect students to submit a revised version of the survey after others had tried to hack it and the rubric would focus on that rather than on all of the steps along the way. I don't feel super strongly about it, but wonder if maybe we should make a time to discuss this week?

**Comment by retabak** [comment]
Interesting. I see value in both ways. I think I'm currently leaning toward wanting to give credit for the collaborative process but could go either way and am happy to revise the rubric if that's what we decide is best. Let's discuss and see how @schanzer feels?

**Comment by flannery-denny** [comment]
I think the thing I’m sure of is that I would change the directions to require revisions (rather than have them talk about revisions). I would want to be assessing them not on the original draft of their survey but of a polished version that’s ready to distribute (hopefully we would actually use it to collect data at this point!), even if the rubric acknowledged process.

If @schanzer is in agreement with that as the end goal, I could take what @retabak’s done and asciidocify it with some modifications for y’all to react to.

Flannery

**Comment by schanzer** [comment]
@flannery-denny definitely in agreement! 

![image](https://user-images.githubusercontent.com/77126/165001811-1bafd2cb-171d-4efa-8a7a-a76e4a562dc5.png)

**Comment by retabak** [comment]
Great!

**Comment by schanzer** [comment]
@flannery-denny I'm happy with the proposed changes. As for data types -- Google Forms allows you to specify "validators", which specify whether an input is a number, an email address, text, etc. You can even say "must be a positive number less than 25", for example. This is described in #4, at the bottom of the lesson itself.

**Comment by retabak** [comment]
I'm loving the changes you made @flannery-denny ! This looks awesome! 

Here's another go at a rubric: https://docs.google.com/document/d/1pPr1OyFcUMWwZ6k5jlLmC4IVatr2ZOQwLIshkOdBtP8/edit

I divided it into two different sections. Let me know what you think.

And, one little thought: If I were teaching this lesson, I think I would have groups make their surveys as a launch to the entire lesson. I think that makes more sense if we're hoping to see them improve how they collect data. I also wonder if some groups will integrate all of the cleanliness mechanisms on their rough drafts, because they participated in the lesson. Then they wouldn't don't benefit from reflecting and discovering on their own? 

Maybe we add a little pedagogy note and suggest that, if teachers would like, they could do the brainstorming / survey creation BEFORE teaching the lesson?

**Comment by flannery-denny** [comment]
@retabak this rubric reads much better to me! Thanks! Curious whether @schanzer would like the titles of the bottom 4 categories to more closely mirror the language from 1-4 in the lesson plan.

**Comment by schanzer** [comment]
@retabak Flannery is right -- the bottom four rows should use language from the lesson plan:

1. Required
2. Question Format (this is a more general form of your "mutliple-choice" row)
3. Descriptive Instructions (this is your "instructions" row)
4. Validation (this is your data type row, but in more detail and not _really_ about data types in the Contract sense of the term)

**Comment by retabak** [comment]
@flannery-denny @schanzer I've pushed an asciidoc of the rubric! Take a look!

Note: I couldn't figure out how to chunk it into two bigger segments like I did on the google doc. (First chunk - did you work with your team? second chunk - did you thoughtfully integrate data cleanliness mechanisms?) @schanzer  - can you help me out with that?

**Comment by schanzer** [comment]
@retabak I don't think this is a good idea -- the rubric is already VERY long (I had to do a lot of language tweaking just to get it to print on one page _without_ the headings), and those headings don't add any useful instructions for students. The items in the first chunk don't really talk about how well teams worked together, and the items in the second chunk are just a more detailed way of asking if students were thoughtful about data hygiene). 

I've fixed the rubric so it all prints on one page and pushed. If you feel strongly about the chunks, let me know.
